src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,44 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_LENGTH = 10;
+    private final Object[] arrayOfKeys = new Object[ARRAY_LENGTH];
+    private final Object[] arrayOfValues = new Object[ARRAY_LENGTH];
+    private int size = 0;
```

Comment:
```suggestion
    private static final int MAX_STORAGE_LENGTH = 10;
    private final Object[] keys = new Object[ARRAY_LENGTH];
    private final Object[] values = new Object[ARRAY_LENGTH];
    private int size;
```

`size` will be initialized by 0 by default